tabs/pid_tuning.js

```diff
@@ -701,11 +718,8 @@ TABS.pid_tuning.initRatesPreview = function () {
     this.keepRendering = true;
     this.model = new Model($('.rates_preview'), $('.rates_preview canvas'));
 
-    var scale = d3.scale.linear().domain([900, 2100]);
-
-    this.rollScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.pitchScale = scale.range([Math.PI * 2, -Math.PI * 2]);
-    this.yawScale = scale.range([Math.PI * 2, -Math.PI * 2]);
+    this.$superExpo = $('.pid_tuning input[name="show_superexpo_rates"]');
```

Comment:
Ah I see. I agree that multi layered closures are a pain, and that they should be avoided and reduced where possible. At the same time, closures **are** a way of defining a private scope, and I do not see this as a bad thing, particularly in an open source, collaborative project like this, where coordination is less strict than in a corporate environment. Members that are added to objects become in a way part of the 'public API', which means that care should be taken to structure them in a way that makes them as generically useful as possible, so that other devs can get value from them as well.